rust/lance/src/index/vector/ivf/io.rs

```diff
@@ -331,8 +325,11 @@ pub(super) async fn write_hnsw_quantization_index_partitions(
         )
         .await?;
 
+        let offset = writer.len();
         if row_id_array.is_empty() {
-            tasks.push(tokio::spawn(async { Ok(0) }));
+            ivf.add_partition(offset, 0);
```

Comment:
in v2, do we track the rows instead of byte offsets in index file?

Reply:
here `offset` is row offset, only IVF_PQ tracks the byte offsets.
all indices will track the row range in v2